### PR TITLE
[hotfix] Update nodes which may have previously shared a piwik_site_id with another node

### PR DESCRIPTION
**Pull request by lyndsysimon**

### CODE REVIEW

File: scripts/migrate_piwik_base_nodes.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+
+def get_nodes():
+    forked = Q('__backrefs.forked.node.forked_from', 'ne', None)
+    registered = Q('__backrefs.registrations.node.registered_from', 'ne', None)
+    templated = Q('__backrefs.template_node.node.template_node', 'ne', None)
+    duplicate = (forked | registered | templated)
+
+    return Node.find(
+        duplicate and Q('date_created', 'lt', datetime.datetime(2014, 10, 31))
```
Comment (sloria) [inline scripts/migrate_piwik_base_nodes.py]: `and` should be `&`

File: scripts/migrate_piwik_base_nodes.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    )
+
+
+def main():
+    init_app('website.settings', set_backends=True, routes=False)
+
+    if 'dry' in sys.argv:
+        if 'list' in sys.argv:
+            logger.info("=== Nodes ===")
```
Comment (sloria) [inline scripts/migrate_piwik_base_nodes.py]: Use singlequotes for strings.

**Comment by lyndsysimon** [comment]
I've no idea why this Travis build failed. @sloria, do you have any insight before I start digging into it?

**Comment by sloria** [comment]
@lyndsysimon The failure is unrelated to your code changes. I've fixed the tests on master.

**Comment by lbanner** [comment]
@sloria -- Deployed to Production yet? TY

**Comment by sloria** [comment]
@lbanner The migration is now in progress.

**Pull request merged**